greenlight/tests/functional/toolbar/test_back_forward.py

```diff
@@ -12,6 +12,14 @@ class TestBackForward(FirefoxTestCase):
     def setUp(self):
         FirefoxTestCase.setUp(self)
 
+        with self.marionette.using_context("content"):
+            self.marionette.navigate("about:blank")
+
+        self.marionette.execute_script("""
+            let count = gBrowser.sessionHistory.count;
+            gBrowser.sessionHistory.PurgeHistory(count);
+        """)
+
```

Comment:
This is not related at all to this PR. Can you explain why this has been added? What this test really would need is to wait for an event, and that module is still to create once I'm done with the observers.